### ISSUE TITLE
nw.js tries to phone home to google

### ISSUE THREAD

**Issue by jonwwilkes** [nwjs/nw.js#5343]
Read the following before submitting your issue, thank you.

nw.js 0.14.7 LTS on Windows 7

About ten seconds after my nw.js-based app starts, Norton security warns me of an outbound tcp connection nw is trying to make to a google server:
origin IP: 192.168.blah.blah (my machine) on port 14089

outbound addy:
clients4.google.com
74.125.21.101:443

**Comment by rogerwang** [comment]
Does this happen with 0.17.6?

**Comment by jonwwilkes** [comment]
I'm not sure... I am using LTS for Windows as an easy way to continue supporting winxp.  But I can try just switching it out with 0.17.6 on the Windows 7 machine and see what happens

**Comment by jonwwilkes** [comment]
> Does this happen with 0.17.6?

Yep.  Happens with 0.17.6 SDK:

Norton gives me a prompt for whether I want nw to have net access for TCP:

192.168.136:-11121 -> clients1.google.com 74.125.138.101:443

**Comment by rogerwang** [comment]
Thanks. With 0.17.6 does it happen right after app start or after a while?

**Comment by rogerwang** [comment]
And since I cannot reproduce it, could you please upload the log when this happens?

https://www.chromium.org/for-testers/enable-logging

**Comment by jonwwilkes** [comment]
Same with 0.17.6-- ten seconds or so.

Sure, I'll try the logging bit when I get a chance.

**Comment by jonwwilkes** [comment]
I'm sorry, but nw.js doesn't actually log anything.  I successfully added `"chromium-args": "--enable-logging --v=1"` to package.json and triggered the issue in my app.  But there is no log file in the relevant directory.  (And I viewed the relevant directories using both the command line and the file browser with "Show Hidden Folders" enabled.)

**Comment by jonwwilkes** [comment]
Oh, and by "relevant" I mean `C:\Users\blah\AppData\Local\Chromium\User Data`.  I see no logs inside there (or any of its subdirs).

**Comment by jonwwilkes** [comment]
Just tried `"chromium-args": "--disable-background-networking"` and it made no difference.  nw still tries to access google over tcp

**Comment by jonwwilkes** [comment]
@rogerwang I figured out a workaround.  I added this to the manifest:
`"chromium-args": "--proxy-server=http://127.0.0.1"`

I don't know if that completely solves it, but I don't get any more notifications of chromium trying to contact google.

However, I am communicating exclusively from the node.js API so I have no need of any Chromium net traffic.  Therefore, I'd urge you to leave this issue open.  Plenty of users of nw.js are just porting web apps to the desktop, and in those cases this probably isn't a workable solution.
